### PR TITLE
Fix encoding

### PR DESCRIPTION
**Pull request by wivarn**

### CODE REVIEW

File: test/consumable_test.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# encoding: utf-8
```
Comment (philkcw) [inline test/consumable_test.rb:1]: What happens if you remove this line?

File: test/consumable_test.rb [hunk 2]
```diff
@@ -35,11 +36,12 @@ def test_consumable
     consumer = Consumer.new
     consumer.subscribe
 
-    msg = Time.now.to_s
+    msg = Time.now.to_s + '™'
```
Comment (philkcw) [inline test/consumable_test.rb:39]: Please:

assert_equal "UTF-8", msg.encoding.upcase

**Comment by philkcw** [comment]
:ok:

**Comment by owenthereal** [comment]
Why is this even an issue? Rails should be using UTF-8 by default

**Comment by wivarn** [comment]
We're getting airbreaks for 
`Encoding::CompatibilityError: incompatible character encodings: ASCII-8BIT and UTF-8`
on this line
https://github.com/jingweno/message_queue/blob/master/lib/message_queue/consumable.rb#L67
When UTF-8 characters are in the payload (user name, org tagline, etc.)

**Comment by wivarn** [comment]
@jingweno But I actually agree with you, this shouldn't even be a problem. I don't know why there is ASCII-8BIT in part of the message.

**Comment by owenthereal** [comment]
@Wivarn I would say check the source of the message. See why it's not forced encoded before passing it to this gem.

**Pull request closed without merging**